tests/mentorship_relation/test_dao_cancel_relation.py

```diff
@@ -105,4 +105,4 @@ def test_dao_mentorship_cancel_relation_not_in_accepted_state(self):
         db.session.commit()
 
         result = DAO.cancel_relation(self.second_user.id, self.mentorship_relation.id)
-        self.assertEqual(({'message': 'This mentorship relation is not in the accepted state.'}, 400), result)
\ No newline at end of file
+        self.assertEqual(({'message': 'This mentorship relation is not in the accepted state.'}, 400), result)
```

Comment:
@LordGameleo try to follow pep8 standards, for example, this line is too long. The line length limit is 79 characters. Thank you!